src/javascript/_common/base/login.js

```diff
@@ -17,7 +17,7 @@ const Login = (() => {
         }
     };
 
-    const loginUrl = () => {
+    const loginUrl = (domain) => {
```

Comment:
```suggestion
    const getLoginUrl = (domain) => {
```

Reply:
It's an old function. I think it’s better we don’t touch it. 
If this is necessary please inform me to change it in whole file.

Reply:
Up to you. I think function names should include **verbs**.

Reply:
I've changed it.